TPP.ArgsParsing.Tests/ArgsParserTest.cs

```diff
@@ -12,21 +12,21 @@ public class ArgsParserTest
         public void TestTooFewArguments()
         {
             var argsParser = new ArgsParser();
-            argsParser.AddArgumentParser(new IntParser());
+            argsParser.AddArgumentParser(new NonnegativeIntParser());
```

Comment:
Unsigned int is a lot better imo, or at least go for absolute int

Reply:
I wouldn't use "unsigned int" because that's misleading. It's still a regular signed int, the nonnegative refers to its validation, not its data type. "absolute int" sounds weird and non-descriptive to me, but maybe I just haven't heard of "absolute" being used in this context yet

Reply:
https://en.wikipedia.org/wiki/Absolute_value